01-python/infrastructure/tree.py

```diff
@@ -1,3 +1,16 @@
 def draw_tree():
-    print("tree not found")
-    return
+    num_trees = 1
```

Comment:
this should be a function parameter.
```suggestion
    def draw_tree(num_trees=1):
```